.travis.yml

```diff
@@ -95,7 +95,7 @@ before_install:
   - git describe --tags --always
   # Required until this is fixed: https://github.com/travis-ci/travis-ci/issues/9112
   - sudo apt-get update --option Acquire::Retries=100 --option Acquire::http::Timeout="60"
-  - pip install -U pip codecov tox
+  - pip install -U pip codecov virtualenv==16.7.9 tox
```

Comment:
what was the error message you were getting?

Reply:
Hi! We got the error in our Travis CI builds, so we don't get to see the full traceback of the error. But here is the error message we got:
```
ERROR:root:ImportError: cannot import name 'ensure_text'
ERROR: InvocationError for command /home/travis/virtualenv/python3.6.3/bin/python -m virtualenv --no-download --python /home/travis/virtualenv/python3.6.3/bin/python pythonlint3 (exited with code 1)
```

We reached out to Travis CI support team. They helped us by running a diff check of builds and found that the only difference is the version of virtualenv. 
Please feel free to let us know if you need any additional information. Thank you!

Reply:
I think installing six>=1.12 would have also solved your issue, even with the latest virtualenv. Though you might want to wait for https://github.com/pypa/virtualenv/pull/1578 to land before you switch.

Reply:
thank you for your suggestion! we used this PR as a temporary workaround just to get our builds pass and will wait for the virtualenv change as you suggested :)